index.bs

```diff
@@ -0,0 +1,331 @@
+          ac.onstatechange = function() {
+            if (ac.state == "running") {
+              // Start running audio app
+            } else {
+              // Display a bit of UI to ask the user to start the audio app.
+              // Audio starts via calling ac.resume() from a handler, and
+              // 'onstatechange' allows knowing when the audio stack is ready.
+            }
+          }
+        }
+      </pre>
+    </div>
+
+  <h3 id="media-element-api">The HTMLMediaElement API</h3>
+    <pre class="idl">
+    partial interface HTMLMediaElement {
+      readonly attribute AutoplayPolicy autoplayPolicy;
+    };
+    </pre>
+    This represents the current status of whether this {{HTMLMediaElement}} is
+    allowed to autoplay or not.
+
+    <dl class="switch">
+      <dt>If the value is {{allowed}}</dt>
+      <dd>
+        This media element is allowed to autoplay.
+      </dd>
+
+      <dt>If the value is {{allowed-muted}}</dt>
+      <dd>
+        This media element will only be allowed to autoplay if it's an
+        {{inaudible media element}}. If authors make this media element audible
+        right after the media element started playing, then the user-agent
+        **MUST** pause this media element immediately because it's no longer
+        inaudible.
+      </dd>
+
+      <dt>If the value is {{disallowed}}</dt>
+      <dd>
+        This media element is not allowed to autoplay. If authors call its
+        {{HTMLMediaElement/play()}}, the returned promise from
+        {{HTMLMediaElement/play()}} will be rejected with {{NotAllowedError}}
+        exception.
+      </dd>
+    </dl>
+
+    If the media element's {{HTMLMediaElement/autoplayPolicy}} is different from
+    the document's {{Document/autoplayPolicy}}, the media element's
+    {{HTMLMediaElement/autoplayPolicy}} overrides the document's
+    {{Document/autoplayPolicy}}.
+
+<h2 id="code-example">Examples</h2>
+  <div class=example>
+    Example of using Document's {{Document/autoplayPolicy}}
+    <pre class="lang-javascript">
+      switch (document.autoplayPolicy) {
+        case "allowed":
+          loadUnmutedVideos();
+          break;
+        case "allowed-muted":
+          loadMutedVideos();
+          break;
+        default:
+          loadPosterImages();
+          break;
+      }
+    </pre>
+  </div>
+
+  <div class=example>
+    Example of using HTMLMediaElement's {{HTMLMediaElement/autoplayPolicy}}
+    <pre class="lang-javascript">
+      function handlePlaySucceeded() {
+        // Update the control UI to playing
+      }
+      function handlePlayFailed() {
+        // Show a button to allow users to explicitly start the video and
+        // display a image element as poster to replace the video
+      }
+
+      let video = document.getElementById("video");
+      switch (video.autoplayPolicy) {
+        case "allowed":
+          video.src = "video.webm";
+          await video.play().then(handlePlaySucceeded, handlePlayFailed);
```

Comment:
```suggestion
          video.play().then(handlePlaySucceeded, handlePlayFailed);
```

I don’t think `await` makes sense here or below if you’re also using `then`?

Reply:
done, will update it later.